finetuning/llama/finetune.py

```diff
@@ -0,0 +1,104 @@
+# finetune.py
```

Comment:
add comment at the top of the file that says how to run this script on beam

Reply:
I added 
"Deploy to beam by running `$ python finetune.py` in the terminal"